tree/treeplayer/inc/ROOT/TDataFrame.hxx

```diff
@@ -244,8 +244,11 @@ const BranchNames &PickBranchNames(unsigned int nArgs, const BranchNames &bl, co
 
 class TDataFrameActionBase {
 protected:
+   std::weak_ptr<ROOT::Detail::TDataFrameImpl> fFirstData;
+   const BranchNames fTmpBranches;
    std::vector<TVBVec_t> fReaderValues;
 public:
+   TDataFrameActionBase(const std::weak_ptr<ROOT::Detail::TDataFrameImpl>& firstData, const BranchNames& bn);
```

Comment:
Note that the conding conventions requires typedef/using to end in _t.  I.e.  BranchNames defines as
```
tree/treeplayer/inc/ROOT/TDataFrame.hxx:using BranchNames = std::vector<std::string>;
```
Thanks,
Philippe.